### DISCUSSION TITLE
maize tassel detection

### DISCUSSION THREAD

**Discussion by stabag19** [geezacoleman/OpenWeedLocator#126] [Q&A]
hi
I am wondering if this can identify a particular part of a plant of maize
I need to identify the male flower 
I read some papers/researches in these last month and it seems possible
I would like to test this owl

**Comment by geezacoleman** [comment]
The biggest barrier here would be the getting the dataset required to train such a model and seeing how well that performs. But if you can train a model like YOLO to do it, then the OWL should be able to run it. Whether or not it's a weed won't matter, you could just use the actuation/outputs to trigger something else that's more useful for your tassel detection.

What would you like to do once they're detected? Let me know how you go testing it.
